src/test/src/controllers/unicorporatedCorrespondenceAddressConfirmController.test.ts

```diff
@@ -0,0 +1,31 @@
+import mocks from "../../mocks/all_middleware_mock";
+import supertest from "supertest";
+import app from "../../../main/app";
+
+import { UNINCORPORATED_CORRESPONDENCE_ADDRESS_CONFIRM, BASE_URL, UNINCORPORATED_SELECT_AML_SUPERVISOR } from "../../../main/types/pageURL";
+
+jest.mock("@companieshouse/api-sdk-node");
+const router = supertest(app);
+
+describe("GET" + UNINCORPORATED_CORRESPONDENCE_ADDRESS_CONFIRM, () => {
+    it("should render the confirmation page with status 200", async () => {
+        const res = await router.get(BASE_URL + UNINCORPORATED_CORRESPONDENCE_ADDRESS_CONFIRM);
+        expect(res.status).toBe(200);
+        expect(mocks.mockSessionMiddleware).toHaveBeenCalled();
+        expect(mocks.mockAuthenticationMiddleware).toHaveBeenCalled();
+    });
```

Comment:
In the test you should check the loaded page is correct using expect(res.text).toContain(Page Heading)